db/seeds.d/101-locations.rb

```diff
@@ -4,17 +4,15 @@
 # !!! PLEASE KEEP THIS SCRIPT IDEMPOTENT !!!
 #
 
-if Location.exists? || ENV['SEED_LOCATION'].present?
```

Comment:
whats the reasoning behind removing this? (just curious)

Reply:
See: https://github.com/Katello/katello/pull/7864

change in foreman.